### PR TITLE
Add ClaimItems action

### PR DESCRIPTION
**Pull request by U-lis**
Make `ClaimItems` action plain value using in-house lib9c module

### CODE REVIEW

**Review by riemannulus — APPROVED** [review]

**Review by ipdae — DISMISSED** [review]

**Comment by riemannulus** [comment]
Could you resolve conflicts?

**Comment by U-lis** [comment]
> Could you resolve conflicts?

@riemannulus Sorry for late reply. I've resolved all conflicts.

File: common/lib9c/actions/__init__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            "values": self._plain_value
+        }
+
+    @property
+    def _plain_value(self):
+        raise NotImplementedError
+
+    @property
+    def serialized_plain_value(self):
+        return bencodex.dumps(self.plain_value)
```
Comment (riemannulus) [inline common/lib9c/actions/__init__.py:25]: If you want to make `ActionBase` serializable, I suggest defining like [`IBencodex` interface](https://github.com/planetarium/bencodex.net/blob/main/Bencodex/IBencodable.cs).

File: common/lib9c/actions/claim_items.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    - values: List[claim_data]
+      - claimData: List[claimItem]
+        - claimItem: Dict[str, Address|List[FungibleAssetValue]]
+          - valid keys: ["avatarAddress", "fungibleAssetValues"]
+      - memo: JSON serialized string
+    """
+
+    def __init__(self, *, claim_data: List[Dict[str, Address | List[FungibleAssetValue]]], memo: Optional[str] = None,
+                 _id: Optional[str] = None):
+        super().__init__("claim_items", _id)
```
Comment (riemannulus) [inline common/lib9c/actions/claim_items.py:22]: How about...

```suggestion
    """
    TYPE_ID: str = "claim_items"

    def __init__(self, *, claim_data: List[Dict[str, Address | List[FungibleAssetValue]]], memo: Optional[str] = None,
                 _id: Optional[str] = None):
        super().__init__(TYPE_ID, _id)
```

**Review by riemannulus — APPROVED** [review]
LGTM

**Pull request merged**